src/devices/LidarLiteV3/samples/README.md

```diff
@@ -0,0 +1,195 @@
+﻿# LidarLiteV3  LIDAR ToF Sensor by Garmin
+
+# Summary
+
+This device belongs to a class of sensors known as time-of-flight, which measures distances by 
+calculating the time delay between signal transmission and reception of the signal as it bounces 
+off the subject. Unlike the popular sonar-based HC-SR04, this device uses a low-power laser. The 
+advantage is longer distances (up to 40m) but more prone to errors when the subject is reflective. 
+
+# Raspberry PI Wiring
+
+The device communicates over I2C and while there's a PWN mode supported by the device, it is not 
+implemented.
+
+I2C Wiring on the Raspberry PI:
+
+| LidarLiteV3 Wire      | Raspberry PI GPIO Pin                   |
+|-----------------------|-----------------------------------------|
+| 5V (red)              | 5V (pin 2)                              |
+| Ground (black)        | Ground (pin 6)                          |
+| Power enable (orange) | Optional, an available IO pin (pin 33)  |
+| Mode control (yellow) | Not used                                |
+| I2C SCL (green)       | SCL (pin 5)                             |
+| I2C SDA (blue)        | SDA (pin 3)                             |
+
+> Important! Don't forget to enable I2C via `raspi-config`.
```

Comment:
:) surprisingly many people forget about it - I hope to eventually add some generated part of the doc which will add some more comprehensive note on at least every I2C/SPI device